lib/locations/locations.go

```diff
@@ -131,9 +136,9 @@ var locations = make(map[LocationEnum]string)
 func expandLocations() error {
 	newLocations := make(map[LocationEnum]string)
 	for key, dir := range locationTemplates {
-		for varName, value := range baseDirs {
-			dir = strings.ReplaceAll(dir, "${"+string(varName)+"}", value)
-		}
+		dir = os.Expand(dir, func(s string) string {
+			return baseDirs[BaseDirEnum(s)]
+		})
```

Comment:
fancy

Reply:
Sorry for being late with another review.  I just noticed this will probably not work with the `${timestamp}` placeholder in the panic and audit logs?

Reply:
Oooh exciting side effect